Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -0,0 +1,150 @@
+**Note:** If you just want to create and share explorations, you may be able to use the hosted server at https://www.oppia.org (in which case you don't need to install anything).
+
+*These installation instructions were last tested on 24 July 2021. For more information on issues that may occasionally arise with the installation process, please contact _vojtech.jelinek@hey.com_ or see the [Troubleshooting](https://github.com/oppia/oppia/wiki/Troubleshooting) page (but that page is not yet updated for Python 3).*
+
+## Downloading Oppia ##
+
+1. Create a new, empty folder called `oppia/` in your computer's home folder. If you already use `oppia/` for Python 2 Oppia feel free to use some different folder name. 
```

Comment:
Add comma after Oppia
some --> a

Reply:
This file change isn't actually part of this PR. I just forgot to push the latest `master` from `oppia/oppia` to the `master` of `U8NWXD/oppia-wiki`. I've fixed this now